lms/envs/common.py

```diff
@@ -1919,6 +1919,9 @@
     'teams',
 
     'xblock_django',
+
+    # edX Proctoring
+    'edx_proctoring',
```

Comment:
as with cms/envs/common.py, I just want to mention that I've been adding these new IDA-style Djangoapps in the OPTIONAL_APPS section, because we can't be sure if they're actually going to be installed or not (at least, that's how we're supposed to view it).  In addition I've been adding client adapters in common/djangoapps/utils -- see milestones_helpers.py, and there will be a new organizations_helpers.py coming along soon, too.